deepsea.spec

```diff
@@ -112,6 +130,20 @@ install -m 644 %{_saltceph}/admin/key/init.sls %{buildroot}/%{_saltceph}/admin/k
 install -d -m 755 %{buildroot}/%{_saltceph}/admin/files
 install -m 644 %{_saltceph}/admin/files/keyring.j2 %{buildroot}/%{_saltceph}/admin/files
 
+install -d -m 755 %{buildroot}/%{_saltceph}/benchmark/cephfs
+install -m 644 %{_saltceph}/benchmark/cleanup.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/cleanup_working_subdir.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/default.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/fio.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/fio_service.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/init.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/mount.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/prepare_master.sls %{buildroot}/%{_saltceph}/benchmark
+install -m 644 %{_saltceph}/benchmark/working_subdir.sls %{buildroot}/%{_saltceph}/benchmark
+
```

Comment:
There are even more errors in the following lines. I' want to do some file re-organisation anyway...will fix this then.